src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/service/RiskService.kt

```diff
@@ -237,7 +244,7 @@ internal class RiskService(
       null
     } else {
       PredictorScore(
-        date = LocalDateTime.parse(riskScoreResponse.completedDate).toLocalDate().toString(),
+        date = LocalDateTime.parse(riskScoreResponse.completedDate ?: DEFAULT_DATE_TIME_FOR_NULL_VALUE).toLocalDate().toString(),
```

Comment:
Don't parse something then turn it back into a string.  Just take it as it is.  I would think we don't need to worry about the default here, it can just be null.  The default is just we are clear about what we want when we sort - sometimes you might want null to represent something in the future, we want it to represent something old.

Reply:
This is done as `completedDate `is LocalDateTime, but the contract with the UI specifies the date here should be in LocalDate format.

Reply:
OK